WordPress/Classes/ViewRelated/Gutenberg/GutenbergMediaPickerHelper.swift

```diff
@@ -135,15 +134,12 @@ extension GutenbergMediaPickerHelper: WPMediaPickerViewControllerDelegate {
     }
 
     func mediaPickerController(_ picker: WPMediaPickerViewController, shouldShowOverlayViewForCellFor asset: WPMediaAsset) -> Bool {
-        picker !== cameraPicker &&
-        asset.exceedsFreeSitesAllowance() &&
-        !post.blog.hasPaidPlan
+        !post.blog.canUploadAsset(asset)
```

Comment:
Do we still need to check if it's not the camera picker here?

Reply:
Yes, thanks for catching this!!